.travis/deploy

```diff
@@ -74,7 +74,7 @@ git commit -m "Refresh site at ${rev}"
 git push -q upstream HEAD:master
 
 # Do a regular checkout and make a dummy commit
-git clone -q git@github.com:qooxdoo/qooxdoo.github.io.git tmp
+git clone --depth 10 -q git@github.com:qooxdoo/qooxdoo.github.io.git tmp
```

Comment:
I know what it means, but in this particular case, what are these `--depth` options accomplishing? What's so far down the tree that we don't need it, and if we don't need it, why is it there?